hw/ip/usbdev/data/usbdev.prj.hjson

```diff
@@ -8,7 +8,7 @@
     dv_doc:             "../doc/dv",
     hw_checklist:       "../doc/checklist",
     sw_checklist:       "/sw/device/lib/dif/dif_usbdev",
-    version:            "0.5",
+    version:            "1.0",
```

Comment:
Are there requirements for declaring a certain version number? Or is this a pre-declaration, the version for the next goal state? Or... it reflects a certain feature set?

I think I'm having trouble understanding a commit that declares the usbdev IP to be at 1.0, but usbdev has had relatively little of the verification work done.

Reply:
This is a pre-declaration, meaning we are working towards version 1.0. since the actual sign-off status is determined by D/V/L stages.

As laid out in [Versioning](https://docs.opentitan.org/doc/project/development_stages/#versioning), < 1.0 versions are used to work towards intermediate goals, where we would like to cycle through the initial D/V stages with the understanding that these stage metrics are temporary, pending a final feature set. Since this is not the case anymore, and we're working towards the tapeout version, this should be increased to 1.0.

Reply:
Heh, that's true. I think this is supposed to be something more like "spec version", rather than "this is released with the following version". Maybe we should tweak a column header in the docs?

Reply:
That is a fair point - we can certainly do that. 
Maybe we should talk about this in the next TC meeting?